src/views/index.js

```diff
@@ -0,0 +1,5 @@
+const ProviderView = require('./ProviderView')
+
+module.exports = {
+  ProviderView
+}
```

Comment:
Seems like `src/views` also conflicts a bit with splitting into multiple packages? I was thinking we were going to do `provider-views` package for the ProviderView and `uppy-server-connect` (draft name) for `RequestClient`?

Reply:
If `RequestClient` and `Provider` are intended for Uppy Server only, I think it would be helpful if their package names reflected that 🤔

Maybe `uppy-server-request` and `uppy-server-provider`.

Reply:
@arturi hmm, so do you suggest I change these folder names? I think the package splitting is still a bit blurry in my head for now and making this type of decision in regard to it feels a bit fragmented for now. Could we leave the renaming till when the package splitting is actually done maybe? So whoever implements it has a clearer vision.

Reply:
All right, I’ll refactor later on 👌